CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 0.4.1 (UNRELEASED)
 
+__BACKWARDS INCOMPATIBILITIES:__
```

Comment:
We are keeping the default false since the volume of metrics Nomad emits is quite large.